thirdparty/DiskANN/CMakeLists.txt

```diff
@@ -238,6 +238,5 @@ include_directories( ${KNOWHERE_SOURCE_DIR}/thirdparty/ )
 add_definitions( -DAUTO_INITIALIZE_EASYLOGGINGPP )
 
 add_subdirectory(src)
-# Don't need executable files
-# add_subdirectory(tests)
-# add_subdirectory(tests/utils)
+add_subdirectory(tests)
+add_subdirectory(tests/utils)
```

Comment:
We do these in two PRs, why don't make a PR delete these(including the test folder), and rebase these two PR on that.